roles/mirror_ocp_release/tasks/facts.yml

```diff
@@ -7,8 +7,9 @@
   when:
     - mor_version is version("4.8", ">=")
   ansible.builtin.shell: >
-    {{ mor_cache_dir }}/{{ mor_version }}/{{ mor_installer }} coreos print-stream-json >
-    {{ mor_cache_dir }}/{{ mor_version }}/rhcos.json
+    flock -x -w 30 "{{ mor_cache_dir }}/{{ mor_version }}/installer.lock" -c
```

Comment:
So, are we finally cover the case of rhcos.json file here? From previous commands, I was supposing that this would not be covered

Reply:
yes, we'll cover it. I was able to reproduce the issue and identified the two tasks that are conflicting.
They are the "Extract installer command from release image" and "Write rhcos.json from command 4.8+" tasks.
I'll update the PR description as Tony requested in the first place.